File: BOJ/Bronze-11050.c
#include <stdio.h>

long long factorial( long long num );
long long combination( long long n, long long k );

int main( void )
{
    long long n, k;
    scanf( "%lld%lld", &n, &k );

    printf( "%lld", combination( n, k ) );

    return 0;
}

long long factorial( long long num )
{
    long long result = 1;
    if ( num == 1 ) return result;
    else
    {
        long long i;
        for ( i = num; i > 0; i-- )
            result = (result * i);
    }

    return result;
}

long long combination( long long n, long long k )
{
    return factorial( n ) / (factorial( k ) * factorial( n-k )); 
}

File: BOJ/Bronze-11050.js
const readline = require('readline');
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

const array = [];
rl.on('line', line => {
  array.push(line);
}).on('close', () => {
// 답안지 시작

const factorial = (num) => {
  if(num == 0) return 1;
  else return num * factorial(num - 1);
}

let result;
const numsArr = array[0].split(' ')
const dif = Number(numsArr[0])-Number(numsArr[1])
if(dif===0) result = 1
else {
  result = Math.round(factorial(Number(numsArr[0]))/(factorial(Number(numsArr[1]))*factorial(dif)))
}
console.log(result);
    
// 답안지 종료
  process.exit();
})